### PR TITLE
fix the bloom and dolly ffn fusion error

### PR DESCRIPTION
**Pull request by intellinjun**
## Type of Change

bug  fix

## Description
The ffn fusion of bloom ,gptneox and dolly  
detail description 
Dolly and GPTNEOX's FFN fusion contains ADD, MPT weight usage errors in FFN calculations

## Expected Behavior & Potential Risk

GPTNEOX and Dolly can output usable sentences, but it will affect performance, and the output of MPT is not correct

### CODE REVIEW

**Comment by intellinjun** [comment]
https://inteltf-jenk.sh.intel.com/job/nlp-toolkit-validation-top-mr-extension/1307/

**Comment by intellinjun** [comment]
image

**Comment by intellinjun** [comment]
image

**Comment by intellinjun** [comment]
image

**Comment by airMeng** [comment]
will you re-submit this pr to [intel:v1.2-rc1](https://github.com/intel/intel-extension-for-transformers/tree/v1.2-rc1) as well?

**Review by a32543254 — APPROVED** [review]
LGTM

**Comment by a32543254** [comment]
> will you re-submit this pr to [intel:v1.2-rc1](https://github.com/intel/intel-extension-for-transformers/tree/v1.2-rc1) as well?

Indeed @intellinjun , please also pick this pr to v1.2-rc1 and co-work with wenxin

File: intel_extension_for_transformers/llm/runtime/graph/application/main_run.cpp [hunk 1]
```diff
@@ -464,9 +464,16 @@ int main(int argc, char** argv) {
         for (model_token token_id = 0; token_id < n_vocab; token_id++) {
           candidates.emplace_back(model_token_data{token_id, logits[token_id], 0.0f});
         }
-
+
```
Comment (airMeng) [inline intel_extension_for_transformers/llm/runtime/graph/application/main_run.cpp:467]: format

Comment (intellinjun) [reply intel_extension_for_transformers/llm/runtime/graph/application/main_run.cpp]: done

**Review by airMeng — APPROVED** [review]

**Pull request merged**